pkg/gen/nodejs-templates/yaml.ts.mustache

```diff
@@ -286,7 +217,87 @@ import * as outputs from "../types/output";
     export class ConfigGroup extends CollectionComponentResource {
         constructor(name: string, config: ConfigGroupOpts, opts?: pulumi.ComponentResourceOptions) {
             super("kubernetes:yaml:ConfigGroup", name, config, opts);
-            this.resources = parse(config, {parent: this});
+
+            let vals: parseInput[] = [];
+            if (config.files) {
+                for (const file of config.files) {
+                    vals.push({name: name, type: "path", value: file})
+                }
+            }
+
+            if (config.yaml) {
+                if (typeof config.yaml === 'string') {
+                    vals.push({name: name, type: "yaml", value: config.yaml});
+                } else {
+                    for (const text of config.yaml) {
+                        vals.push({name: name, type: "yaml", value: text})
+                    }
+                }
+            }
+
+            const promise = pulumi.output(pulumi.runtime.invoke(
+                "kubernetes:yaml:parse", {input: vals}, {async: true}));
+
+            this.resources = promise.apply(p => {
+                let rs = {};
+                Object.keys(p.result).forEach((name: string) => {
+                    Object.keys(p.result[name]).forEach((path: string) => {
+                        if (path.startsWith("__yaml")) {
+                            const resources = parseYamlDocument({
+                                    objs: p.result[name][path],
+                                    transformations: config && config.transformations || [],
+                                    resourcePrefix: config && config.resourcePrefix,
+                                },
```

Comment:
Why is this duped between `yaml.ts` and `yaml.ts.mustache`?

Reply:
`yaml.ts` is generated from `yaml.ts.mustache`